docs/api/text/tsne.rst

```diff
@@ -20,20 +20,30 @@ After importing the required tools, we can :doc:`load the corpus <corpus>` and v
     corpus = load_corpus('hobbies')
     tfidf  = TfidfVectorizer()
 
-    docs   = tfidf.fit_transform(corpus.data)
-    labels = corpus.target
+    X   = tfidf.fit_transform(corpus.data)
+    y   = corpus.target
 
 Now that the corpus is vectorized we can visualize it, showing the distribution of classes.
 
 .. code:: python
 
     # Create the visualizer and draw the vectors
     tsne = TSNEVisualizer()
-    tsne.fit(docs, labels)
+    tsne.fit(X, y)
     tsne.poof()
 
 .. image:: images/tsne_all_docs.png
 
+Now,let's pass on a list of labels to TSNEVisualizer.
+
+.. code:: python
+
+    #Pass on a list of labels
+    labels = corpus.categories
+    tsne = TSNEVisualizer(labels=labels)
+    tsne.fit(X, y)
+    tsne.poof()
```

Comment:
This certainly implements the change requested by the issue by creating another code snippet that uses the labels argument. However, because this code would draw the same exact figure as the code above, might I suggest we move forward with one of two possible revisions? 

The first option is to simply refactor this into a note, something like:

```
Note that you can pass the class labels or document categories directly to the ``TSNEVisualizer`` as follows:

.. code:: python

    # Pass a list of class labels for the legend
    labels = corpus.categories 
    tsne = TSNEVisualizer(labels=labels)
```

This is a little more compact, and also note the use of the double backticks (` ``TSNEVisualizer`` `) which indicate this is a piece of code. 

The second option is to update the clustering code snippet below on LIne 69 as follows:

```
tsne = TSNEVisualizer(labels=["c{}".format(c) for c in range(5)])
tsne.fit(X, clusters.labels_)
```

You can then add some text to the line above or below that reflect the use of the `labels` argument as in the previous text. 

What do you think of these ideas, do you think one of them is more clear than the other?

Reply:
I will change the docs as instructed.Thank you for your patience and guidance.

Reply:
I think both are okay.We should keep both the formats so as the other contributors can learn to write a piece of code in those formats.Both are basic python statements,and it will be fun to know both when playing around with code.